apps/src/panels/PanelsView.tsx

```diff
@@ -208,6 +210,7 @@ const PanelsView: React.FunctionComponent<PanelsProps> = ({
       >
         {showButton && (
           <Button
+            key={'button-' + currentPanelIndex}
```

Comment:
both this line and line 159 really should be written as template strings, as preferred modern syntax.

e.g., `button-${currentPanelIndex}`.

There's also an eslint rule that would enforce it (and I'm pretty sure also automatically clean it up), if we wanted to enable it.

Reply:
Ah good point.  Might do this more broadly in a follow-up!

Reply:
Fixed in https://github.com/code-dot-org/code-dot-org/pull/62208.